packages/paste-website/src/components/DoDont.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+const StyledWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-column-gap: 24px;
+`;
+
+interface DoDontProps {
+  children?: React.ReactNode;
+}
+
+function DoDont(props: DoDontProps) {
+  return <StyledWrapper>{props.children}</StyledWrapper>;
+}
+
+const StyledChild = styled.div``;
```

Comment:
Maybe these could be called something based on `Example`? `StyledExample`, `StyledExampleImg`, `StyledExampleContent`, `StyledExampleTitle` etc

Reply:
Good call, I will get that updated 👍